bundles/net.tourbook/src/net/tourbook/map2/view/SlideoutMap2_TourColors.java

```diff
@@ -55,6 +55,7 @@ public class SlideoutMap2_TourColors extends ToolbarSlideout implements IActionR
    private Button   _chkShowInChartToolbar_Altitude;
    private Button   _chkShowInChartToolbar_Gradient;
    private Button   _chkShowInChartToolbar_Pace;
+   private Button   _chkShowInChartToolbar_Power;
```

Comment:
This comment is for the following line in this file

` * Copyright (C) 2005, 2021 Wolfgang Schramm and Contributors`

When you modify a file you may also check, that the copyright year is updated. This is not dramatically and I also forget it sometimes.